packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.tsx

```diff
@@ -0,0 +1,165 @@
+    const { refreshNotifications } = notifications;
+
+    const is_mx_mlt = landing_company_shortcode === 'iom' || landing_company_shortcode === 'malta';
+
+    React.useEffect(() => {
+        if (!is_switching) {
+            WS.authorized.getAccountStatus().then((response: AccountStatusResponse) => {
+                const { get_account_status } = response;
+                if (get_account_status) {
+                    const {
+                        allow_document_upload,
+                        allow_poa_resubmission,
+                        needs_poi,
+                        needs_poa,
+                        document_status,
+                        is_age_verified,
+                        poa_address_mismatch,
+                    } = populateVerificationStatus(get_account_status);
+                    const has_submitted_poa = document_status === PoaStatusCodes.pending && !allow_poa_resubmission;
+
+                    setAuthenticationStatus({
+                        ...authentication_status,
+                        allow_document_upload,
+                        allow_poa_resubmission,
+                        needs_poi,
+                        needs_poa,
+                        document_status,
+                        has_submitted_poa,
+                        is_age_verified,
+                        poa_address_mismatch,
+                    });
+                    setIsLoading(false);
+                    refreshNotifications();
+                }
+            });
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_switching]);
+
+    const handleResubmit = () => {
+        setAuthenticationStatus({ ...authentication_status, ...{ resubmit_poa: true } });
+    };
+
+    const onSubmit = (needs_poi: boolean) => {
+        setAuthenticationStatus({ ...authentication_status, ...{ has_submitted_poa: true, needs_poi } });
+    };
+
+    const {
+        allow_document_upload,
+        allow_poa_resubmission,
+        document_status,
+        needs_poi,
+        resubmit_poa,
+        has_submitted_poa,
+        is_age_verified,
+        poa_address_mismatch,
+    } = authentication_status;
+
+    const from_platform = getPlatformRedirect(app_routing_history);
+
+    const should_show_redirect_btn = Object.keys(platforms).includes(from_platform?.ref ?? '');
+
+    const redirect_button = should_show_redirect_btn ? (
+        <Button
+            primary
+            className='proof-of-identity__redirect'
+            onClick={() => {
+                const url = platforms[from_platform.ref as keyof typeof platforms]?.url;
+                if (url) {
+                    window.location.href = url;
+                    window.sessionStorage.removeItem('config.platform');
+                }
+            }}
+        >
+            <Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name: from_platform.name }} />
+        </Button>
+    ) : null;
```

Comment:
```suggestion
    const redirect_button = should_show_redirect_btn && (
        <Button
            primary
            className='proof-of-identity__redirect'
            onClick={() => {
                const url = platforms[from_platform.ref as keyof typeof platforms]?.url;
                if (url) {
                    window.location.href = url;
                    window.sessionStorage.removeItem('config.platform');
                }
            }}
        >
            <Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name: from_platform.name }} />
        </Button>
    );
```

Reply:
This change is going to make the type of the redirect button prop in the components to be either `JXS.Element or false`. But It has to be `JSX.Element or null`